deploy/basic/terraform/variables.tf

```diff
@@ -192,3 +192,80 @@ locals {
   compute_shape_description              = lookup(local.compute_shape_flexible_vs_descriptions, local.instance_shape, local.instance_shape)
   lb_shape_flexible                      = "flexible"
 }
+
+# Additional stuff for extra demo (DNS, Cert, WAF, Traffic)
+# DNS Entries
+variable "enable_dns_zone" {
+  default = false
+}
+
+variable "dns_lb_entry" {
+  default = "internal-lb"
+}
+
+variable "dns_waf_entry" {
+  default = "store"
+}
+
+locals {
+  dns_lb_domain = "${var.dns_lb_entry}.${var.dns_zone_name}"
+  dns_waf_domain = "${var.dns_waf_entry}.${var.dns_zone_name}"
+}
+
+# Certificate
+variable "certificate_certificate_name" {
+  default = "mushop-lets-encrypt-certificate"
+}
+
+variable "enable_acme_certificate" {
+  default = false
+}
+
+variable "acme_email" {
+  default = ""
+}
+
+variable "acme_server_url" {
+  default = "https://acme-staging-v02.api.letsencrypt.org/directory"
```

Comment:
Using staging for production?

Reply:
I could switch that so the staging one in the variables file needs to be commented out, but Lets Encrypt uses recommends using non-prod until you are ready, due to API usage limits.  I will have a look at it.

Reply:
Yes, you can have an option for staging, but if some will be using a FQDN hostname, standing will not generate valid certificates for any browser.